packages/plugin-ext/src/plugin/preferences/configuration.ts

```diff
@@ -143,6 +143,9 @@ export class ConfigurationModel {
 
     private mergeContents(source: any, target: any): void {
         for (const key of Object.keys(target)) {
+            if (key === '__proto__') {
```

Comment:
Should we test for `prototype` here?

Reply:
I saw that as well: https://github.com/lodash/lodash/pull/4336/files
so I would say `__proto__` is enough

Reply:
Ok